web/backend/index.js

```diff
@@ -0,0 +1,186 @@
+const express = require('express');
+const path = require('path');
+const axios = require('axios');
+const cookieParser = require("cookie-parser");
+const session = require('express-session');
+//import kyber from "@dedis/kyber";
+const kyber = require('@dedis/kyber')
+const crypto = require('crypto');
+const fetch = require('node-fetch');
+const request = require('request');
+
+
+const test = kyber.curve.newCurve('edwards25519');
+
+const app = express();
+
+// Serve the static files from the React app
+app.use(express.static(path.join(__dirname, 'client/build')));
+
+//Express-session
+app.set('trust-proxy', 1);
+
+app.use(cookieParser());
+const oneDay = 1000 * 60 * 60 * 24;
+app.use(session({
+    secret: "thisismysecrctekeyfhrgfgrfrty84fwir767",
+    saveUninitialized:true,
+    cookie: { maxAge: oneDay },
+    resave: false
+}));
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
+//serving public file
+//app.use(express.static(__dirname));
+
+
+// An api endpoint that returns a short list of items
+app.get('/api/getTkKey', (req,res) => {
+
+    body = 'urlaccess=https://dvoting-dev.dedis.ch/api/control_key\nservice=Evoting\nrequest=name,firstname,email,uniqueid,allunits';
+    axios
+        .post('http://tequila.epfl.ch/cgi-bin/tequila/createrequest', body)
+        .then(resa => {
+            key = resa.data.split('\n')[0].split('=')[1];
+            url = 'https://tequila.epfl.ch/cgi-bin/tequila/requestauth?requestkey=' + key;
+            res.json({'url' : url});
+        })
+        .catch(error => {
+            console.error(error)
+        });
+});
+
+app.get('/api/control_key', (req, res) => {
+    usr_key = req.query.key;
+    body = 'key=' + usr_key;
+
+    axios
```

Comment:
you're using `fetch` in the frontend so I would stay with that and not add another dependency.